crates/oxc_semantic/src/binder.rs

```diff
@@ -20,16 +20,16 @@ pub trait Binder<'a> {
 impl<'a> Binder<'a> for VariableDeclarator<'a> {
     fn bind(&self, builder: &mut SemanticBuilder<'a>) {
         let (includes, excludes) = match self.kind {
-            VariableDeclarationKind::Const => (
+            VariableDeclarationKind::Using
+            | VariableDeclarationKind::AwaitUsing
+            | VariableDeclarationKind::Const => (
```

Comment:
Complete nit, but I suggest listing `Const` first is preferable - the others are pretty obscure.

```suggestion
            VariableDeclarationKind::Const
            | VariableDeclarationKind::Using
            | VariableDeclarationKind::AwaitUsing => (
```

Reply:
It would assume it will compile down to exactly same assembly, so 0 perf impact. My suggestion was purely on "code style" grounds.